packages/react/src/patterns/sub-patterns/PictogramItem/README.md

```diff
@@ -0,0 +1,87 @@
+# Pictogram Item
+
+## Getting started
+
+Here's a quick example to get you started.
+
+```scss
+// yourapplication.scss
+@import '@carbon/type/scss/font-face/mono';
+@import '@carbon/type/scss/font-face/sans';
+@include carbon--font-face-mono();
+@include carbon--font-face-sans();
+@import '@carbon/ibmdotcom-styles/scss/patterns/sub-patterns/pictogram-item';
+```
+
+> 💡 Only import font's once per usage
+
+```javascript
+import { Desktop } from '@carbon/pictograms-react';
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { PictogramItem } from '@carbon/ibmdotcom-react';
+import 'yourapplication.scss';
+
+function App() {
+  const heading = 'Lorem ipsum dolor sit amet.';
+  const copy = 'Lorem ipsum dolor sit amet, consectetur adipiscing elit.';
+
+  const cta = {
+    type: 'text',
+    href: 'https://www.example.com',
+    copy: 'Lorem ipsum dolor',
+  };
+
+  return (
+    <PictogramItem
+      heading={heading}
+      copy={copy}
+      cta={cta}
+      Pictogram={Desktop}
+    />
+  );
+}
+
+ReactDOM.render(<App />, document.querySelector('#app'));
+```
+
+> 💡 Don't forget to import the pictogram item styles from
+> [@carbon/ibmdotcom-styles](https://github.com/carbon-design-system/ibm-dotcom-library/blob/master/packages/styles).
+
+## Props
+
+| Name        | Required | Data Type | Default Value | Description                                                          |
+| ----------- | -------- | --------- | ------------- | -------------------------------------------------------------------- |
+| `heading`   | YES      | String    | null          | Heading text                                                         |
+| `copy`      | YES      | String    | null          | Copy text                                                            |
+| `Pictogram` | YES      | Component | null          | Pictogram component imported from `@carbon/pictograms-react` package |
+| `cta`       | NO       | Object    | null          | Object with cta data, see `cta` bellow                               |
+
+### cta
+
+| Name   | Data Type | Description   |
+| ------ | --------- | ------------- |
+| `type` | String    | CTA Type      |
+| `href` | String    | CTA href      |
+| `copy` | String    | CTA copy text |
```

Comment:
Wonder if we need to spell it out, or if we can just link them to the text link cta option for more information.

Reply:
maybe as we'll be only dealing with href and copy this would be better to have just these 2 options on the documentation?

Reply:
Only problem I see with that is if it is defined by another component `LinkWithIicon` those props could change, and then now we have to track down everywhere it's mentioned like here in the documentation instead of having one source of truth and updating it there.

Reply:
makes sense, I'll link the CTA documentation, thanks!

Reply:
@photodow I've placed a link to the path where the CTA documentation will be available, once the block is finished.

Reply:
We should probably be linking to the CTA README file, that's still in PR though with @mkothur .

Reply:
but yes, in general I agree with @photodow , we are following this same methodology with our functional specs where we reference other components/patterns/etc, we should do the same here in our READMEs.

Reply:
@jeffchew  I've linked the readme file from Mahesh PR.